modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -52,133 +87,174 @@ handled by the codemod are marked with 🤖 in the Upgrade Guide.**
+`type` attribute was not being set which resulted in the buttons
+[defaulting to `type="submit"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attributes).
 
-We found an issue where the default action for `form` controls like `input` and `textarea` is to
-click the first `submit` `button` which in some cases is not a `submit` button.
+This affects all buttons which extend `BaseButton`:
 
-An example of this causing issues was our `Modal.CloseIcon` and `Modal.CloseButton`. When the
-`Modal` is wrapped in a `form` element, the close icon would submit the `form`.
+- `PrimaryButton`
+- `SecondaryButton`
+- `TertiaryButton`
+- `DeleteButton`
+- `ToolbarDropdownButton` and `ToolbarIconButton`
+- `Pill` and `Pill.IconButton`
+- `SegmentedControl.Button` (Main)
+- `SegmentedControl.Item` (Preview)
+- `Pagination.PageButton`
 
-This change affects all of our `buttons` including:
-`PrimaryButton, SecondaryButton, TertiaryButton, ToolbarButton`.
+Any buttons which extend any of the above are affected as well.
 
-If you still wish for a specific `button` within a `form` to submit, just add the attribute
-`type="submit"`.
+This resolves an [issue](https://github.com/Workday/canvas-kit/issues/1938) where clicking certain
+buttons within a `form` element would unexpectedly submit the form. Additionally, the default action
+of many form controls such as `input` and `textarea` is to click the first submit button; this led
+to issues if the first submit button in the form was not intended to be a submit button (a common
+mistake when buttons default to `type="submit"`).
+
+This is a breaking change if you expected a button to submit by default, though we do not anticipate
+this to be the case for most developers. If you _do_ intend for a button to act as a submit button
+within a form, add `type="submit"` to the button.
+
+---
 
 ### Toast
 
-We've promoted `Toast` from Labs (`@workday/canvas-kit-labs-react`) to the Main
-(`@workday/canvas-kit-react`) package.
+**PR:** [#2044](https://github.com/Workday/canvas-kit/pull/2044)
+
+We've promoted `Toast` from Labs to Main. It replaces the `Toast` that was previously in Main (for
+reference, see
+[`Toast`](https://d2krrudi3mmzzw.cloudfront.net/v8/?path=/docs/components-popups-toast--error) in
+Main from v8).
 
-- This component is a
-  [compound component](/getting-started/for-developers/resources/compound-components/).
-- It has a `mode` prop that allows for proper accessibility attributes to be defined.
-- Its sub components allow for more flexibility and access to lower level elements.
+`Toast` is a [compound component](/getting-started/for-developers/resources/compound-components/)
+which provides a flexible API and access to its internals via its subcomponents. It supports a new
+`mode` prop which applies the proper accessibility features to the component based on the desired
+mode: `status`, `alert`, or `dialog`.
```

Comment:
I don't know how to word this but we don't provide a system for toasts, is that worth mentioning. That's always been the case so maybe it's ok to omit.

Reply:
I know you said you weren't sure how to word it, but can you provide more context on what you mean by a "system for toasts"?

Reply:
Like we just provide the toast card. Its' up to the consumer to hook up everything with popups and the right aria live regions. It's kinda out of scope for this.

Reply:
Got it. Yeah, if anything, this sort of content would belong in the Toast component documentation. It could go in the Upgrade Guide, but I don't think it's necessary.